vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/gridConnector.js

```diff
@@ -944,6 +944,11 @@ window.Vaadin.Flow.gridConnector = {
     function _fireClickEvent(event, eventName) {
       if (grid.$connector.activeItem) {
         event.itemKey = grid.$connector.activeItem.key;
+        const eventContext = grid.getEventContext(event);
+        // if you have a details-renderer, getEventContext().column is undefined
+        if (eventContext.column) {
+          event.flowKey = eventContext.column._flowId;
```

Comment:
Just an internal naming thing, but I'd like to see all variables referring to the internal column id be called `columnId` instead of `flowKey`, `flowColumnKey` etc. This makes it easier to read the code and understand that it's the thing that is used in `idToColumnMap`. Using "key" can cause confusion, since column-keys are the things that can be set by user. And like I mentioned earlier, the "flow"-part is used just for the web component's property name to avoid naming conflicts in the wild west of JavaScript.

Reply:
I used `flowId` - `columnId` sounds to me like a thing you can set with `column.setId`, where as `flowId` indicates it is Flow's internal. Also change the new method in Grid to be `getColumnByFlowId` to make it more consistent. Does this make sense?

Reply:
Well, I still prefer my idea. I might be biased because I'm used to working with Grid internals and I know there are these internal ids, so I don't get confused with `column.setId`.

If you'd look for the first time at the arguments of `new ItemClickEvent()` and see there `String flowId`, would you understand based on this name what it is? Id of Flow framework? There's nothing in that name indicating that it's actually column-specific identifier.

I think the term "internal" is key here, not "flow". So if `columnId` gets confused with `column.setId`, IMO the best option would be `columnInternalId` or `internalColumnId`.

Reply:
`internalColumnId` it is then :)